scripts/tests/run_test_suite.py

```diff
@@ -114,11 +121,16 @@ class RunContext:
     help='Internal flag for running inside a unshared environment'
 )
 @click.option(
-    '--manual',
-    type=click.Choice(['skip', 'only', 'include'], case_sensitive=False),
-    default='skip',
-    show_default=True,
-    help='Internal flag to determine how to handle manual tests. ONLY for "all" test choice.',
+    '--include-tags',
+    type=click.Choice(TestTag.__members__.keys(), case_sensitive=False),
+    multiple=True,
+    help='What test tags to include when running.',
+)
+@click.option(
+    '--exclude-tags',
+    type=click.Choice(TestTag.__members__.keys(), case_sensitive=False),
+    multiple=True,
+    help='What test tags to exclude when running.',
```

Comment:
Should probably document whether include or exclude takes priority if some test matches both.